src/components/accession2/view/DetailPanel.tsx

```diff
@@ -99,7 +99,8 @@ export default function DetailPanel(props: DetailPanelProps): JSX.Element {
     const subdivision = accession?.collectionSiteCountrySubdivision;
     const data = [
       city,
-      countries && country && subdivision ? getSubdivisionByCode(countries, country, subdivision)?.name : '',
+      (countries && country && subdivision && getSubdivisionByCode(countries, country, subdivision)?.name) ??
+        subdivision,
```

Comment:
do we need another `?? ''` after subdivision on line 103 (in case we already have accessions with an undefined subdivision)

Reply:
Not necessary, just tested and if subdivision is undefined, the location shows the country only